community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/schema/SchemaDescriptor.java

```diff
@@ -34,8 +35,15 @@
  * how this is done in eg. LabelSchemaDescriptor, and the SchemaProcessor and SchemaComputer interfaces need to be
  * extended with methods taking the new concrete type as argument.
  */
-public interface SchemaDescriptor
+public interface SchemaDescriptor extends SchemaDescriptorSupplier
 {
+    boolean isAffected( long[] entityIds );
+
+    enum PropertySchemaType
```

Comment:
would be nice to get javadoc for this class since the purpose of this enum is not really clear